pix-editor/tests/acceptance/pix-readonly-test.js

```diff
@@ -0,0 +1,29 @@
+import { module, test } from 'qunit';
```

Comment:
L'objectif de ce test d'acceptance peut être de tester la navigation dans les frameworks. Du coup, perso :
- Je renommerai le fichier en `acceptance/navigate-through-frameworks_test.js` (par exemple)
- Je mettrai en nom de module `module('Acceptance | Navigate through frameworks')`
- Je ferai autant de tests que de rôles, style `when user role is <role>, it should display selected framework` ou le contraire si le role est restreint.
- Enfin, mais ça c'est super bonus, je me demande si on peut pas mettre la ember-testing-library de pix sur le projet PE pour rendre les tests d'acceptance plus lisibles/simples